### ISSUE TITLE
About - One Blog | 文鹃阁

### ISSUE THREAD

**Issue by yarkable** [yarkable/yarkable.github.io#3]
https://yarkable.github.io/about/ 

 Plz go to study now !

**Comment by JiananYuan** [comment]
tql%%%%，您是szu CS的研究生吗？

**Comment by yarkable** [comment]
是的哈哈

**Comment by yarkable** [comment]
> @JiananYuan
> tql%%%%，您是szu CS的研究生吗？

是的哈哈

**Comment by yarkable** [comment]
> @JiananYuan
> tql%%%%，您是szu CS的研究生吗？

大佬更强！膜

**Comment by JiananYuan** [comment]
大佬，我对您下面这个python爬虫项目很感兴趣：
”szu-newsboard-spider
深圳大学公文通爬虫“
请问把这个项目如果挂在服务器上，是不是需要在服务器上挂校内vpn，进公文通需要身份验证。

**Comment by JiananYuan** [comment]
大佬，我对您下面这个python爬虫项目很感兴趣：
”szu-newsboard-spider
深圳大学公文通爬虫“
请问把这个项目如果挂在服务器上，是不是需要在服务器上挂校内vpn，进公文通需要身份验证。(服务器上挂校内vpn应该不会自动掉线吧)

**Comment by yarkable** [comment]
> @JiananYuan
> 大佬，我对您下面这个python爬虫项目很感兴趣：
> ”szu-newsboard-spider
> 深圳大学公文通爬虫“
> 请问把这个项目如果挂在服务器上，是不是需要在服务器上挂校内vpn，进公文通需要身份验证。(服务器上挂校内vpn应该不会自动掉线吧)
是的 如果挂在校外服务器上的话是需要VPN回内网的，之前Linux服务器我没找到合适的方案，后来协会成员在Windows server用easyconnect连回学校就行了

**Comment by yarkable** [comment]
@JiananYuan 是的 如果挂在校外服务器上的话是需要VPN回内网的，之前Linux服务器我没找到合适的方案，后来协会成员在Windows server用easyconnect连回学校就行了

**Comment by stdAri** [comment]
请问，如果方便的话可否添加一个RSS订阅的链接，

**Comment by yarkable** [comment]
@stdAri 好的，忙完论文就开搞

**Comment by yarkable** [comment]
> @stdAri
> 请问，如果方便的话可否添加一个RSS订阅的链接，

已添加，欢迎订阅！

**Comment by Bartender-bit** [comment]
大佬您好，可以请教下您在“https://github.com/BingfengYan/CO-MOT/issues/18”这个issue提到的多个不同ID框套在同一目标上的问题是怎么解决的吗，我也遇到了这个问题QAQ

**Comment by yarkable** [comment]
> 大佬您好，可以请教下您在“[https://github.com/BingfengYan/CO-MOT/issues/18”这个issue提到的多个不同ID框套在同一目标上的问题是怎么解决的吗，我也遇到了这个问题QAQ](https://github.com/BingfengYan/CO-MOT/issues/18%E2%80%9D%E8%BF%99%E4%B8%AAissue%E6%8F%90%E5%88%B0%E7%9A%84%E5%A4%9A%E4%B8%AA%E4%B8%8D%E5%90%8CID%E6%A1%86%E5%A5%97%E5%9C%A8%E5%90%8C%E4%B8%80%E7%9B%AE%E6%A0%87%E4%B8%8A%E7%9A%84%E9%97%AE%E9%A2%98%E6%98%AF%E6%80%8E%E4%B9%88%E8%A7%A3%E5%86%B3%E7%9A%84%E5%90%97%EF%BC%8C%E6%88%91%E4%B9%9F%E9%81%87%E5%88%B0%E4%BA%86%E8%BF%99%E4%B8%AA%E9%97%AE%E9%A2%98QAQ)

具体我也忘了怎么解决的了，根本原因是没训好，可以尝试多加点数据，后面我把视频数据加进去好像就ok了

**Comment by Bartender-bit** [comment]
> @yarkable
> > 大佬您好，可以请教下您在“[https://github.com/BingfengYan/CO-MOT/issues/18”这个issue提到的多个不同ID框套在同一目标上的问题是怎么解决的吗，我也遇到了这个问题QAQ](https://github.com/BingfengYan/CO-MOT/issues/18%E2%80%9D%E8%BF%99%E4%B8%AAissue%E6%8F%90%E5%88%B0%E7%9A%84%E5%A4%9A%E4%B8%AA%E4%B8%8D%E5%90%8CID%E6%A1%86%E5%A5%97%E5%9C%A8%E5%90%8C%E4%B8%80%E7%9B%AE%E6%A0%87%E4%B8%8A%E7%9A%84%E9%97%AE%E9%A2%98%E6%98%AF%E6%80%8E%E4%B9%88%E8%A7%A3%E5%86%B3%E7%9A%84%E5%90%97%EF%BC%8C%E6%88%91%E4%B9%9F%E9%81%87%E5%88%B0%E4%BA%86%E8%BF%99%E4%B8%AA%E9%97%AE%E9%A2%98QAQ)
> 
> 具体我也忘了怎么解决的了，根本原因是没训好，可以尝试多加点数据，后面我把视频数据加进去好像就ok了

好的感谢！
